src/wp-content/themes/twentynineteen/style-editor.css

```diff
@@ -1605,3 +1605,8 @@ ul.wp-block-archives li ul,
   padding-right: 0;
   width: 100%;
 }
+
+/* Post Author Block  */
+.editor-styles-wrapper .avatar, .editor-styles-wrapper .wp-block-post-author__avatar img {
+  border-radius: 50%;
+}
```

Comment:
The CSS will auto generate based on SCSS when it build, Could you please remove this changes?